sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -730,6 +729,13 @@ trait CheckAnalysis extends PredicateHelper with LookupCatalog {
     }
   }
 
+  private def canonicalizeForError(expr: LogicalPlan): String =
+    if (SQLConf.get.getConf(SQLConf.CANONICALIZE_PLANS_IN_ERRORS)) {
```

Comment:
Actually we don't need such a new config. We can just check 
```
Utils.isTesting
```

Reply:
@gengliangwang thanks for the suggestion! This is done, it is simpler now.